app/views/ctc/questions/income/edit.html.erb

```diff
@@ -9,18 +9,3 @@
     <% end %>
   </ul>
 <% end %>
-
-<% content_for :script do %>
```

Comment:
Don't need this anymore because we're picking up the timezone on the efile security information, and then grabbing from there to set on the intake.